common/src/main/java/org/conscrypt/ConscryptEngine.java

```diff
@@ -210,6 +212,14 @@ private static NativeSsl newSsl(SSLParametersImpl sslParameters, ConscryptEngine
         }
     }
 
+    /**
+     * Configures the default {@link BufferAllocator} to be used by all future
+     * {@link SSLEngine} instances from this provider.
+     */
+    public static void setDefaultBufferAllocator(BufferAllocator bufferAllocator) {
```

Comment:
This one shouldn't be public.

Reply:
Poor copy/paste on my part, apologies.
Fixed